build.gradle

```diff
@@ -1,10 +1,10 @@
 plugins {
-    id 'java'
-    id 'groovy'
-    id 'maven'
-    id 'maven-publish'
-    id 'idea'
-    id 'eclipse'
+    id("java")
+    id("groovy")
```

Comment:
We're not actually using groovy for this plugin, so this should be safe to remove.

Reply:
reverted that to groovy style and groovy strings